ui/README.md

```diff
@@ -59,7 +59,12 @@ want to run them from the local install using one of:
 
 If you're adding an npm dependency, you'll need to run:
```

Comment:
I think this can do with a bit more rewording. How about:


To modify an existing `npm` dependency, you'll need to edit `package.json` in the standard fashion. To add a new `npm` dependency, you'll want to run:
```shell
npm install --save <dep>
```
Either way, complete the process by running:
```shell
rm -r node_modules && npm update && node_modules/.bin/shonkwrap
```
Be sure to commit any changes to `shrinkwrap.json`.